webplugin/js/app/km-utils.js

```diff
@@ -50,12 +50,12 @@ KommunicateConstants = {
+        5: '<svg width="17" height="18" viewBox="0 0 17 18" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M6.93761 11.4307H10.0626C10.3473 11.4307 10.5834 11.1946 10.5834 10.9098C10.5834 10.6251 10.3473 10.389 10.0626 10.389H6.93761C6.65289 10.389 6.41678 10.6251 6.41678 10.9098C6.41678 11.1946 6.65289 11.4307 6.93761 11.4307Z" fill="#ED9D00"/><path d="M10.9307 8.30567C11.506 8.30567 11.9723 7.83929 11.9723 7.264C11.9723 6.6887 11.506 6.22233 10.9307 6.22233C10.3554 6.22233 9.889 6.6887 9.889 7.264C9.889 7.83929 10.3554 8.30567 10.9307 8.30567Z" fill="#ED9D00"/><path d="M6.06955 8.30567C6.64485 8.30567 7.11122 7.83929 7.11122 7.264C7.11122 6.6887 6.64485 6.22233 6.06955 6.22233C5.49426 6.22233 5.02789 6.6887 5.02789 7.264C5.02789 7.83929 5.49426 8.30567 6.06955 8.30567Z" fill="#ED9D00"/><path d="M8.49317 2.05566C4.65983 2.05566 1.55566 5.16678 1.55566 9.00011C1.55566 12.8334 4.65983 15.9446 8.49317 15.9446C12.3334 15.9446 15.4446 12.8334 15.4446 9.00011C15.4446 5.16678 12.3334 2.05566 8.49317 2.05566ZM8.50011 14.5557C5.43066 14.5557 2.94455 12.0696 2.94455 9.00011C2.94455 5.93066 5.43066 3.44455 8.50011 3.44455C11.5696 3.44455 14.0557 5.93066 14.0557 9.00011C14.0557 12.0696 11.5696 14.5557 8.50011 14.5557Z" fill="#ED9D00"/></svg>',
+        10: '<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M11.4307 8.30567C12.006 8.30567 12.4723 7.83929 12.4723 7.264C12.4723 6.6887 12.006 6.22233 11.4307 6.22233C10.8554 6.22233 10.389 6.6887 10.389 7.264C10.389 7.83929 10.8554 8.30567 11.4307 8.30567Z" fill="#2EA63A"/><path d="M6.56955 8.30567C7.14485 8.30567 7.61122 7.83929 7.61122 7.264C7.61122 6.6887 7.14485 6.22233 6.56955 6.22233C5.99426 6.22233 5.52789 6.6887 5.52789 7.264C5.52789 7.83929 5.99426 8.30567 6.56955 8.30567Z" fill="#2EA63A"/><path d="M9.00011 11.7779C7.97233 11.7779 7.09039 11.2154 6.60428 10.389H5.44455C6.00011 11.8126 7.38205 12.8196 9.00011 12.8196C10.6182 12.8196 12.0001 11.8126 12.5557 10.389H11.3959C10.9098 11.2154 10.0279 11.7779 9.00011 11.7779ZM8.99317 2.05566C5.15983 2.05566 2.05566 5.16678 2.05566 9.00011C2.05566 12.8334 5.15983 15.9446 8.99317 15.9446C12.8334 15.9446 15.9446 12.8334 15.9446 9.00011C15.9446 5.16678 12.8334 2.05566 8.99317 2.05566ZM9.00011 14.5557C5.93066 14.5557 3.44455 12.0696 3.44455 9.00011C3.44455 5.93066 5.93066 3.44455 9.00011 3.44455C12.0696 3.44455 14.5557 5.93066 14.5557 9.00011C14.5557 12.0696 12.0696 14.5557 9.00011 14.5557Z" fill="#2EA63A"/></svg>',
     },
     RATING_TITLE: {
-        1: "Poor",
```

Comment:
this we need to change for km-server, dashboard, weekly report cron as well. any issues filled for that?